code/modules/modular_computers/file_system/programs/card.dm

```diff
@@ -224,10 +224,10 @@
 						return
 
 					target_id_card.access -= get_all_accesses()
-					target_id_card.access += jobdatum.get_access()
+					target_id_card.access |= jobdatum.get_access()
```

Comment:
is this a copied list?

Reply:
![image](https://user-images.githubusercontent.com/87972842/235105810-d5a74262-cbe9-47e6-91b6-1a02e413ff2a.png)